packages/cdk/lib/cloudquery/index.ts

```diff
@@ -365,16 +372,55 @@ export function addCloudqueryEcsCluster(
 	/*
 	This is a catch-all task, collecting all other AWS data.
 	Although we're not using the data for any particular reason, it is still useful to have.
-		
+
+	After switching to the allow list the table list was too long (with 757 the TaskDefinition was 74049 bytes long.
+	This exceeded the ECS limit of 65536 bytes and meant the stack failed to deploy.
+    We therefore had to split the AwsRemainingData task.
+
 	It runs once a week because there is a lot of data, and we need to avoid overlapping invocations.
 	If we identify a table that needs to be updated more often, we should create a dedicated task for it.
 	*/
-	const remainingAwsSources: CloudquerySource = {
-		name: 'AwsRemainingData',
-		description: 'Data fetched across all accounts in the organisation.',
+
+	const remainingAwsTables = filterAllowedTables(awsTables, [/^aws_.*$/]);
+	const halfOfRemainingAwsTablesNumber = Math.floor(
+		remainingAwsTables.length / 2,
+	);
+
+	const remainingAwsSourcesPart1: CloudquerySource = {
+		name: 'AwsRemainingDataPart1',
+		description: 'Data fetched across all accounts in the organisation part 1.',
+		schedule: Schedule.cron({ minute: '0', hour: '16', weekDay: 'SAT' }), // Every Saturday, at 4PM UTC
+		config: awsSourceConfigForOrganisation({
+			tables: remainingAwsTables.slice(0, halfOfRemainingAwsTablesNumber),
+			//ServiceCatalogue collects child tables automatically.
```

Comment:
```suggestion
			// CloudQuery collects child tables automatically.
```

I'm not sure this comment is too useful here - might it be better as a doc string for the [`skipTables` property](https://github.com/guardian/service-catalogue/blob/0ad898e32d9160ba31f4d4689e2b7a1342cd09f2/packages/cdk/lib/cloudquery/config.ts#L17)?